### PR TITLE
 Format, center content w/ flex, link styles & cta btn from nodejs.org

### PR DESCRIPTION
**Pull request by MattIPv4**
This pull request intends to make the following changes:
 - Format source code to be easier to read & maintain
 - Center the page content using flex
 - Add inline link styling from nodejs.org
 - Add CTA button, following the styling of nodejs.org

View the new design live at https://mattipv4.github.io/node.js.org/

| Before | After |
|--------|------|
| image | image |

### CODE REVIEW

**Comment by MattIPv4** [comment]
cc @Trott

**Comment by Trott** [comment]
LGTM. Anyone from @nodejs/website want to look this over before I merge it?

**Review by Trott — APPROVED** [review]

File: 404.html [hunk 1]
```diff
@@ -4,24 +4,144 @@
...
+            </g>
+            <g clip-path="url(#e)" transform="translate(-78.306 -164.016)">
+                <path fill="url(#f)" d="M197.02 225.934v107.43h91.683v-107.43zm0 0"/>
+            </g>
+        </svg>
     </a>
-  </main>
+    <span>Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.</span>
+    <span>For more information, go to <a href="https://nodejs.org/">https://nodejs.org/</a>.</span>
+    <span><a href="https://nodejs.org" class="button">
```
Comment (Trott) [inline 404.html]: Optional nit-pick: Add a trailing slash to the URL here for consistency with elsewhere in the file.

File: index.html [hunk 2]
```diff
@@ -1,24 +1,144 @@
...
+            </g>
+            <g clip-path="url(#e)" transform="translate(-78.306 -164.016)">
+                <path fill="url(#f)" d="M197.02 225.934v107.43h91.683v-107.43zm0 0"/>
+            </g>
+        </svg>
     </a>
-  </main>
+    <span>Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.</span>
+    <span>For more information, go to <a href="https://nodejs.org/">https://nodejs.org/</a>.</span>
+    <span><a href="https://nodejs.org" class="button">
```
Comment (Trott) [inline index.html]: Repeated optional nit-pick: Add a trailing slash to the URL here for consistency with elsewhere in the file.

**Comment by Trott** [comment]
@MattIPv4 Thanks for doing this. This is definitely an improvement. 😀

File: 404.html (hunk 1 above)
Comment (MattIPv4) [reply 404.html]: Fixed.

File: index.html (hunk 2 above)
Comment (MattIPv4) [reply index.html]: Fixed.

**Review by aymen94 — APPROVED** [review]

**Pull request merged**